feat(curriculum): add second flexbox workshop to FSD cert (#62735)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/blocks/workshop-colorful-boxes/68f91df5776e693049fb8942.md

---
id: 68f91df5776e693049fb8942
title: Step 3
challengeType: 0
dashedName: step-3
---

# --description--

Now go to the `styles.css` file and create a `h1` element. In previous lessons, you learned how to add CSS properties and values like this:

```css
element {
 property: value;
}
```

> Center the content of the `h1` element by setting its `text-align` property with the value `center`. Also create a `margin-bottom` property with the value `10px` to add margin between the bottom of the `h1` element and any element that goes bellow.

Reviewer: ```suggestion
Then center the content of the `h1` element by setting its `text-align` property to `center`. 

Also set the `margin-bottom` property to `10px`. This will push everything below the `h1` element down 10 pixels.
```

# --hints--

> Create a `h1` element

Reviewer: ```suggestion
You should have an `h1` selector.
```

```js
assert.exists(new __helpers.CSSHelp(document).getStyle('h1'));
```

> Add `text-align` property with the value `center`

Reviewer: Could we also write: "Your `h1` CSS rule ..."?
Or even just: "Your `h1` rule ..."?

```js
assert.equal(new __helpers.CSSHelp(document).getStyle('h1')?.getPropVal('text-align'), 'center');
```

> Add `margin-bottom` property with the value `10px`

Reviewer: ```suggestion
Your `h1` selector should have a `margin-bottom` property set to `10px`.
```

---
id: 68f91df5776e693049fb8942
title: Step 3
challengeType: 0
dashedName: step-3
---

# --description--

Now it is time to add some CSS rules to the `styles.css` file. Start by creating a selector for the `h1` element. 

In previous lessons, you learned that the basic syntax looks like this:

```css
element {
 property: value;
}
```

> 

Author: Acknowledged

Then center the content of the `h1` element by setting its `text-align` property to `center`. Also create a `margin-bottom` property with the value `10px` to add margin between the bottom of the `h1` element and any HTML element that goes bellow.

# --hints--

You should have an `h1` element.

```js
assert.exists(new __helpers.CSSHelp(document).getStyle('h1'));
```

Your `h1` element should have a `text-align` property set to `center`.

```js
assert.equal(new __helpers.CSSHelp(document).getStyle('h1')?.getPropVal('text-align'), 'center');
```

> Your `h1` element should have a `margin-bottom` property set to `10px`.

Reviewer 2: ```suggestion
Your `h1` selector should set the `margin-bottom` property to `10px`.
```
